### PR TITLE
Update vs-code.md

### LINKED ISSUE #4150: 'Visual Studio Code' page issue
Page URL: https://flutter.dev/docs/development/tools/vs-code.html
Page source: https://github.com/flutter/website/tree/master/src/docs/development/tools/vs-code.md

Description of issue:
Can we add in details of how to target different flutter build modes in VS Code - debug / release / profile on this page. I couldn't find specific documentation - though it is mentioned briefly here: https://flutter.dev/docs/perf/rendering/ui-performance#run-in-profile-mode

### PR DESCRIPTION
**Pull request by micpap25**
Fixes #4150

Changes proposed in this pull request:

*  Add a section about targeting build modes for VS Code

### CODE REVIEW

File: src/docs/development/tools/vs-code.md [hunk 1]
```diff
@@ -145,6 +145,32 @@ running or debugging.
...
+      
+      
+### Run app in debug, profile, or release mode
+
+Flutter offers many different build modes to run your app in. 
+You can read more about them in [Flutter's build modes][].
+
+ 1. Open the `launch.json` file in VS Code.
+     * If you do not have a `launch.json` file, go to 
+     the Run view in VS Code and click **create a launch.json file**.
```
Comment (sfshaza2) [inline src/docs/development/tools/vs-code.md:157]: => Bold the "Run" text. (Bold menu items:  https://developers.google.com/style/ui-elements)

File: src/docs/development/tools/vs-code.md [hunk 2]
```diff
@@ -145,6 +145,32 @@ running or debugging.
...
+ 1. In the `configurations` section, change the `flutterMode` property to 
+ the build mode you want to target. 
+     * For example, if you want to run in debug mode, 
+     your `launch.json` may look like this:
```
Comment (sfshaza2) [inline src/docs/development/tools/vs-code.md:161]: may => might. (Avoid "may": https://developers.google.com/style/word-list#may)

Also, we don't use bulleted lists for formatting. To have the "For example, ..." to break on a new line, just put a <br> at the end of the previous line. (Also, a one-item list is discouraged...)

File: src/docs/development/tools/vs-code.md [hunk 3]
```diff
@@ -145,6 +145,32 @@ running or debugging.
...
+      "configurations": [
+       {
+         "name": "Flutter",
+         "request": "launch",
+         "type": "dart",
+         "flutterMode": "debug"
+       }
+     ]
+     ```
+ 1. Run the app through the Run view.
```
Comment (sfshaza2) [inline src/docs/development/tools/vs-code.md:172]: => Run the app through the **Run** view.

**Review by sfshaza2 — CHANGES_REQUESTED** [review]
Overall this is really good. Just a couple small things, since you create quite a few PRs.  :)

Also, I staged your PR here:  https://sz-flutter.firebaseapp.com/docs/development/tools/vs-code#run-app-in-debug-profile-or-release-mode

**Comment by micpap25** [comment]
Should be fixed now! I'll keep those style guides on-hand from now on, haha.

**Review by sfshaza2 — APPROVED** [review]
LGTM

**Pull request merged**